pay-api/src/pay_api/services/paybc_service.py

```diff
@@ -205,8 +205,8 @@ def __create_site(self, access_token, party, account, account_info):
             'account_number': account.get('account_number', None),
```

Comment:
Do `account_number` and `customer_name` need to be changed as well?

Reply:
That comes from the account dict (which is a response from PayBC in snake_case).